client/src/infrastructure/getNamespace.js

```diff
@@ -50,24 +50,29 @@ class sdkNamespace {
     return format.formatNamespaces(namespaceList)
   }
 
-  static getNamespaceInfoByName = async name => {
+  static getNamespaceInfo = async name => {
+
+    // TODO: Read params if hex, convert to uint64 id
     const namespace = new NamespaceId(name)
     const namespaceService = new NamespaceService(NAMESPACE_HTTP)
-    let namespaceInfo = await namespaceService.namespace(namespace).toPromise()
 
-    let namespaceNames = await NAMESPACE_HTTP.getNamespacesName([namespace]).toPromise()
+    // let namespaceInfo = await namespaceService.namespace(namespace).toPromise()
+    let namespaceInfo = rootNamespace
+
+    // let namespaceNames = await NAMESPACE_HTTP.getNamespacesName([namespace]).toPromise()
+    let namespaceNames = [namespaceName]
```

Comment:
method receives `name` not `namespaceName`

Reply:
`namespaceName` is DTO and `import { Endpoint, rootNamespace, subNamespace, namespaceName } from '../config/'`